data_sync/mvds.md

```diff
@@ -0,0 +1,97 @@
+# Minimum Viable Data Synchronization
+
+**DRAFT VERSION 1.0**
+
+*Written by Oskar Thorén oskar@status.im & Dean Eigenmann dean@status.im*
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+2. [Wire Protocol](#wire-protocol)
+    1. [Payloads](#payloads)
+3. [Synchronization](#synchronization)
+    1. [State](#state)
+    2. [Flow](#flow)
+    3. [Retransmission](#retransmission)
+4. [Footnotes](#footnotes)
+
+## Abstract
+
+In this paper we describe and specify a minimum viable protocol for data synchronization built on top of the Bramble Synchronization Protocol<sup>1</sup>. This protocol is built on top of the Bramble Synchronization Protocol and was designed in order to ensure reliable messaging between peers communicating across an unreliable peer-to-peer (P2P) network where they may be unreachable or unresponsive. 
```

Comment:
paper or spec?

Reply:
Also probably want to specify things NOT in scope, like mostly-offline, etc

Also where does MVDSRequrements fit into this? Do we want to add these?

What about definitions such as PeerID etc?